packages/web-runtime/src/components/Topbar/Notifications.vue

```diff
@@ -291,11 +292,32 @@ export default {
       setAdditionalData()
     }
 
-    onMounted(() => {
+    const setupSSE = async () => {
```

Comment:
Considering we will use that more often in the future, I think it would be great to have this as a composable in `web-pkg`. You'd then pass the URL and 2 (optional) callbacks for `onopen` and `onmessage`.

Reply:
alright